map.json

```diff
@@ -0,0 +1,155 @@
+  ],
+  "src/ResourceManager/ContainerRegistry/": [
+    ".\\src\\ResourceManager\\ContainerRegistry\\Commands.ContainerRegistry.Test\\bin\\Debug\\Microsoft.Azure.Commands.ContainerRegistry.Test.dll"
+  ],
+  "src/ResourceManager/DataFactories/": [
+    ".\\src\\ResourceManager\\DataFactories\\Commands.DataFactories.Test\\bin\\Debug\\Microsoft.Azure.Commands.DataFactories.Test.dll"
+  ],
+  "src/ResourceManager/DataLakeAnalytics/": [
+    ".\\src\\ResourceManager\\DataLakeAnalytics\\Commands.DataLakeAnalytics.Test\\bin\\Debug\\Microsoft.Azure.Commands.DataLakeAnalytics.Test.dll"
+  ],
+  "src/ResourceManager/DataLakeStore/": [
+    ".\\src\\ResourceManager\\DataLakeStore\\Commands.DataLakeStore.Test\\bin\\Debug\\Microsoft.Azure.Commands.DataLake.Test.dll"
+  ],
+  "src/ResourceManager/DevTestLabs/": [],
+  "src/ResourceManager/Dns/": [
+    ".\\src\\ResourceManager\\Dns\\Commands.Dns.Test\\bin\\Debug\\Microsoft.Azure.Commands.Dns.Test.dll"
+  ],
+  "src/ResourceManager/EventHub/": [
+    ".\\src\\ResourceManager\\EventHub\\Commands.EventHub.Test\\bin\\Debug\\Microsoft.Azure.Commands.EventHubs.Test.dll"
+  ],
+  "src/ResourceManager/HDInsight/": [
+    ".\\src\\ResourceManager\\HDInsight\\Commands.HDInsight.Test\\bin\\Debug\\Commands.HDInsight.Test.dll"
+  ],
+  "src/ResourceManager/Insights/": [
+    ".\\src\\ResourceManager\\Insights\\Commands.Insights.Test\\bin\\Debug\\Microsoft.Azure.Commands.Insights.Test.dll"
+  ],
+  "src/ResourceManager/IotHub/": [
+    ".\\src\\ResourceManager\\IotHub\\Commands.IotHub.Test\\bin\\Debug\\Microsoft.Azure.Commands.IotHub.Test.dll"
+  ],
+  "src/ResourceManager/KeyVault/": [
+    ".\\src\\ResourceManager\\KeyVault\\Commands.KeyVault.Test\\bin\\Debug\\Microsoft.Azure.Commands.KeyVault.Test.dll"
+  ],
+  "src/ResourceManager/LogicApp/": [
+    ".\\src\\ResourceManager\\LogicApp\\Commands.LogicApp.Test\\bin\\Debug\\Microsoft.Azure.Commands.LogicApp.Test.dll"
+  ],
+  "src/ResourceManager/MachineLearning/": [],
+  "src/ResourceManager/Media/": [],
+  "src/ResourceManager/Network/": [
+    ".\\src\\ResourceManager\\Network\\Commands.Network.Test\\bin\\Debug\\Microsoft.Azure.Commands.Network.Test.dll"
+  ],
+  "src/ResourceManager/NotificationHubs /": [
+    ".\\src\\ResourceManager\\NotificationHubs\\Commands.NotificationHubs.Test\\bin\\Debug\\Microsoft.Azure.Commands.NotificationHubs.Test.dll"
+  ],
+  "src/ResourceManager/OperationalInsights/": [
+    ".\\src\\ResourceManager\\OperationalInsights\\Commands.OperationalInsights.Test\\bin\\Debug\\Microsoft.Azure.Commands.OperationalInsights.Test.dll"
+  ],
+  "src/ResourceManager/PowerBIEmbedded/": [
+    ".\\src\\ResourceManager\\PowerBIEmbedded\\Commands.PowerBIEmbedded.Test\\bin\\Debug\\Microsoft.Azure.Commands.PowerBIEmbedded.Test.dll"
+  ],
+  "src/ResourceManager/Profile/": [
+    ".\\src\\ResourceManager\\Profile\\Commands.Profile.Test\\bin\\Debug\\Microsoft.Azure.Commands.Profile.Test.dll"
```

Comment:
Profile changes should result in all tests being run.  Again, I would follow the pattern of using the test directory to make sure this test dll is included